test_linux.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+timeout 20s ./zubax-kucher
```

Comment:
There is no point in just starting the program. The objective is to test its functionality. Run the unit tests here.

Reply:
Come to think of it, just starting the program is not a useless test either. Keep it and run the unit tests, too.